go/vt/sqlparser/ast.go

```diff
@@ -1032,6 +1033,9 @@ const (
 // Format formats the node.
 func (node *ComparisonExpr) Format(buf *TrackedBuffer) {
 	buf.Myprintf("%v %s %v", node.Left, node.Operator, node.Right)
+	if node.Escape != nil {
+		buf.Myprintf(" escape %v", node.Escape)
+	}
```

Comment:
WalkSubtree needs to cover this new node. You may also have to verify that it doesn't crash if the expression is nil.